bin/crew

```diff
@@ -434,9 +434,14 @@ def update
   abort "'crew update' is used to update crew itself. Use 'crew upgrade <package1> [<package2> ...]' to update specific packages.".orange if @pkg_name
 
   # update package lists
+  @other_arches = %w[i686 x86_64 armv7l]
+  @other_arches.reject! { |n| n == ARCH }
   Dir.chdir(CREW_LIB_PATH) do
     # Set sparse-checkout folders.
-    system "git sparse-checkout set packages manifest/#{ARCH} lib commands bin crew tests tools"
+    system "git sparse-checkout set --no-cone \"/*\" bin commands crew lib \"/manifest/#{ARCH}/*\" packages tests tools"
```

Comment:
`--no-clone`?

Reply:
No it's `cone` referring to the new sparse algorithm we are avoiding.

Reply:
https://stackoverflow.com/questions/9572407/git-sparse-checkout-with-exclusion#26129711